src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -202,4 +203,18 @@ static boolean hasOcr(CaseDetails theCase) {
             .map(o -> Validation.<String, Instant>valid(Instant.parse((String) o)))
             .orElse(invalid("Missing " + dateField));
     }
+
+    /**
+     * Used in createCase callback only. Can be reviewed later and with BPS-746 - revised the usage.
+     * @param theCase from CCD.
+     * @return Validation of it which is always valid
```

Comment:
in the callback I don't remember ccd sends an empty list or null value if the OCR data is not present. Should we not fail of scanOCRData is empty or null or there would be nothing to be transformed later

Reply:
Fair point. OK, will restrict this to have ocr strictly